File: LeetCode-Programs/Check if All A's Appears Before All B's.cpp
class Solution {
public:
    bool checkString(string s) {
        for(int i=0;i<s.length();i++){
            if (s[i]=='b'){
                for(int j=i+1;j<s.length();j++){
                    if(s[j]=='a')
                        return false;
                }
            }
            else
                continue;
        }
        return true;
    }
};

File: LeetCode-Programs/Check if All A's Appears Before All B's.py
class Solution:
    def checkString(self, s: str) -> bool:
        for i in range(len(s)):
            if s[i]=='b':
                for j in range(i+1, len(s)):
                    if s[j]=='a':
                        return False
            else:
                continue
        return True
